scripts/linux/trcE2ETest.sh

```diff
@@ -50,6 +50,7 @@ function usage() {
     echo ' -listenWorkloadUri                       Customize listen workload socket'
     echo ' -desiredModulesToRestartCSV              CSV string of module names for long haul specifying what modules to restart. If specified, then "restartIntervalInMins" must be specified as well.'
     echo ' -restartIntervalInMins                   Value for long haul specifying how often a random module will restart. If specified, then "desiredModulesToRestartCSV" must be specified as well.'
+    echo ' -sendReportFrequency                     Value for long haul specifying how often TRC will send reports to LogAnalytics.'
```

Comment:
nit: Looking back, I think we added too many configurable options to the pipelines which we never really change. The downside of this is that we need to maintain a list of variables in each pipeline we create. I feel it could be better to avoid making things configurable at the pipeline level and just rely on defaults in this script, or just hardcode it in the deployment files. 

Even if we still have this configurable from the pipeline level, a default might be a good idea anyways. Your call.

Reply:
There's a default within the dotnet code. So if the parameter is not filled in by the pipeline, it will take the default. We can remove the stuff from the yamls if we want. I'm just going to keep it for now. It's pretty simple to remove if it's annoying us later

Reply:
I think that if the parameter is not filled in by the pipeline, it won't take the dotnet default. This is because in our deployment template we give the value a placeholder. If the pipeline doesn't fill in the placeholder I would expect an exception from the dotnet app.

Reply:
Yeah you're right